### PR TITLE
Translucent bar

### PR DESCRIPTION
**Pull request by sebastianludwig**
Please take a look. This is a barebone implementation for #11 and #12.

### CODE REVIEW

File: Classes/JLNRBarBackgroundView.m [hunk 1]
```diff
@@ -9,6 +9,21 @@
...
 
 @implementation JLNRBarBackgroundView
+{
+    UIView *blurEffectView;
+}
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+    if (self = [super initWithFrame:frame]) {
+        if ([[[UIDevice currentDevice] systemVersion] compare:@"8.0" options:NSNumericSearch] != NSOrderedAscending) {
```
Comment (jlnr) [inline Classes/JLNRBarBackgroundView.m]: I'd prefer `if (NSClassFromString(@"UIBlurEffect"))`

File: Classes/JLNRBarView.h [hunk 2]
```diff
@@ -27,6 +27,8 @@
 @property(nonatomic, getter=isBottomBarHidden) BOOL bottomBarHidden;
 @property(nonatomic, getter=isSideBarHidden) BOOL sideBarHidden;
 
+@property (nonatomic, getter=isBottomBarTranslucent) BOOL bottomBarTranslucent;
+
 - (void)reloadData;
 
 - (void)setBarHidden:(BOOL)hidden animated:(BOOL)animated;
```
Comment (jlnr) [inline Classes/JLNRBarView.h]: Would `setBarsHidden:` be more clear or more confusing?

File: Example/StylingViewController.m [hunk 3]
```diff
@@ -24,6 +24,7 @@ @interface StylingViewController ()
 @property (weak, nonatomic) IBOutlet UISegmentedControl *inactiveControl;
 @property (weak, nonatomic) IBOutlet UILabel *footnoteLabel;
 @property (weak, nonatomic) IBOutlet UISwitch *hiddenSwitch;
+@property (weak, nonatomic) IBOutlet UISwitch *tranclucencySwitch;
```
Comment (jlnr) [inline Example/StylingViewController.m]: typo

File: Example/StylingViewController.m [hunk 4]
```diff
@@ -47,6 +48,11 @@ - (void)viewWillAppear:(BOOL)animated
     
     self.animationSwitch.on = animationsEnabled;
     
+    self.tranclucencySwitch.on = [self jlnrBarController].barView.bottomBarTranslucent;
+    if ([[[UIDevice currentDevice] systemVersion] floatValue] < 8) {
```
Comment (jlnr) [inline Example/StylingViewController.m]: `NSClassFromString` again

Comment (sebastianludwig) [inline Example/StylingViewController.m]: Looking only at the Example, it's pretty much out of context why one would check for `UIVisualEffectView`.

File: Classes/JLNRBarView.h (hunk 2 above)
Comment (sebastianludwig) [inline Classes/JLNRBarView.h]: Not sure. Would you call the property `barsHidden` with a `areBarsHidden` getter then too?

Comment (jlnr) [inline Classes/JLNRBarView.h]: There are two methods starting with `are*` in UIKit, but neither is a property getter (one requires a parameter, one is a class method). So even though there is no real precedent in the Apple world, I would say yes.

Comment (jlnr) [inline Classes/JLNRBarView.h]: But the bigger question is if there is only one bar, conceptually, or if there are two. If there is only one bar, then `isBottomBarHidden` and `isSideBarHidden` feel wrong to me.
Anyway, not an issue with this PR

File: Classes/JLNRBarBackgroundView.m [hunk 5]
```diff
@@ -17,6 +32,18 @@ - (void)setBorderColor:(UIColor *)borderColor
     [self setNeedsDisplay];
 }
 
+- (void)setTranslucent:(BOOL)translucent
+{
+    _translucent = translucent;
+    
+    if (translucent) {
+        blurEffectView.frame = self.bounds;
+        [self insertSubview:blurEffectView atIndex:0];
```
Comment (jlnr) [inline Classes/JLNRBarBackgroundView.m]: The parameter to `insertSubview:` is tagged as `nonnull`, but `blurEffectView` will be `nil` on iOS 7, or when the view is (very unexpectedly) instantiated via `initWithCoder:`

**Comment by jlnr** [comment]
Will merge and fix `insertSubview:`

**Pull request merged**